src/components/admin/order/realtime/modal/order-by-product/OrderByProductModal.tsx

```diff
@@ -0,0 +1,129 @@
+import { Order } from '@@types/index';
+import { colFlex, rowFlex } from '@styles/flexStyles';
+import styled from '@emotion/styled';
+import _ from 'lodash';
+import { useRecoilValue } from 'recoil';
+import { productsAtom } from '@recoils/atoms';
+import { tabletMediaQuery } from '@styles/globalStyles';
+import AppLabel from '@components/common/label/AppLabel';
+import { Color } from '@resources/colors';
+
+const ModalContainer = styled.div`
+  ${colFlex({ align: 'center' })};
+  position: fixed;
+  top: calc(50% - 275px);
+  right: 150px;
+  background-color: rgba(233, 233, 233, 0.8);
+  border-radius: 10px;
+  width: 240px;
+  height: 550px;
+  z-index: 1999;
+  padding: 25px;
+  box-sizing: border-box;
+  gap: 40px;
+
+  ${tabletMediaQuery} {
+    right: 120px;
+  }
+`;
+
+const HeaderContainer = styled.div`
+  ${colFlex({ justify: 'center', align: 'center' })};
+  width: 100%;
+  gap: 3px;
+`;
+
+const ProductContainer = styled.div`
+  ${colFlex()};
+  width: 100%;
+  height: 450px;
+  overflow-y: auto;
+  background: ${Color.LIGHT_GREY};
+  border-radius: 10px;
+  box-sizing: border-box;
+  padding: 10px;
+  gap: 12px;
+`;
+
+const ProductItem = styled.div`
+  ${rowFlex({ align: 'center' })};
+  gap: 20px;
+`;
+
+const ProductImage = styled.img`
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+`;
+
+const ProductInfoContainer = styled.div`
+  ${colFlex({ justify: 'center', align: 'flex-start' })};
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  word-break: break-all;
+`;
```

Comment:
```suggestion
const ProductInfoContainer = styled.div`
  ${colFlex({ justify: 'center', align: 'flex-start' })};
  min-width: 0;

  & > * {
    overflow: hidden;
    white-space: nowrap;
    text-overflow: ellipsis;
    width: 100%;
  }
`;
```
이렇게 적용해야지 밑줄임표가 제대로 적용됩니다.